### PR TITLE
add authorized_three_scale_accounts table

### PR DESCRIPTION
**Pull request by ravsa**

### CODE REVIEW

**Comment by msrb** [comment]
Question: What service(s) will use this table?

**Comment by miteshvp** [comment]
> Question: What service(s) will use this table?

RACHEL integrations @msrb

File: alembic/versions/14a551f25764_three_scale_accounts_table.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    """Upgrade the database to a newer revision."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('authorized_three_scale_accounts',
+                    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+                    sa.Column('account_id', sa.String(length=255), nullable=False),
```
[the remark this answers is not in the document]
Comment (ravsa) [reply alembic/versions/14a551f25764_three_scale_accounts_table.py]: sure @samuzzal-choudhury

**Comment by msrb** [comment]
@miteshvp @samuzzal-choudhury 

> RACHEL integrations @msrb

Thanks, I read that as "not workers" :)

Here's the thing:

Ingestion RDS database is, in my opinion, not the right place for these tables. I know it's already quite messy there, but it doesn't mean we should add more unrelated tables. This PR will make any RACHEL-related service/deployment dependent on workers. Such tight coupling will make development, deployment and testing unnecessarily more difficult. Developers who will want to fix a bug or implement new features to the RACHEL-related service will, from some reason, need to know how to deploy ingestion subsystem.

Much cleaner way would be to create a special RACHEL database and keep the tables there. There can be multiple databases in the same RDS instance. Of course, the RACHEL service would be responsible for the database migrations.

**Comment by samuzzal-choudhury** [comment]
[test]

**Comment by samuzzal-choudhury** [comment]
> Ingestion RDS database is, in my opinion, not the right place for these tables. I know it's already quite messy there, but it doesn't mean we should add more unrelated tables. This PR will make any RACHEL-related service/deployment dependent on workers.

@msrb This new table is a facilitator of authorizing requests for the coreapi functionalities. IMHO, it can be included in the existing database with the same reason for including `stack_analyses_request`.

@msrb @miteshvp do we already have a plan to identify and create a separate database for the non-worker tables from the existing database?

**Comment by samuzzal-choudhury** [comment]
[test]

**Pull request closed without merging**